graas-bt.py

```diff
@@ -246,138 +220,6 @@ def read_gps_data(ser):
 
     return gps
 
-def handleButton(text):
```

Comment:
I'm seeing a lot of code removed here without being replaced elsewhere - just double checking that that's ok